### PR TITLE
Db request improvement, sink store improvement, othere fixes

### PR DESCRIPTION
**Pull request by VSemikozInostudio**

### CODE REVIEW

File: lib/src/store/moor/moor_store.dart [hunk 1]
```diff
@@ -397,6 +417,152 @@ class MoorStore extends Store {
     return _processRoomRecords(roomRecords, timelineLimit, memberIds, context);
   }
 
+  //TODO rename
```
Comment (MaginMaxim) [inline lib/src/store/moor/moor_store.dart]: Do we need this TODO comment?

File: lib/src/store/moor/moor_store.dart [hunk 2]
```diff
@@ -397,6 +417,152 @@ class MoorStore extends Store {
...
+          upgrade: record.upgradeRecord?.toRoomEvent(),
+        ),
+        timeline: timeline,
+        memberTimeline: memberTimeline,
+        summary: RoomSummary(
+          joinedMembersCount: roomRecord.summaryJoinedMembersCount,
+          invitedMembersCount: roomRecord.summaryInvitedMembersCount,
+        ),
+        directUserId: roomRecord.directUserId != null
+            ? UserId(roomRecord.directUserId ?? '')
```
Comment (MaginMaxim) [inline lib/src/store/moor/moor_store.dart]: Why do we use ?? check, if we have already checked, that value != null?

File: lib/src/store/moor/moor_store.dart (hunk 1 above)
Comment (VSemikozInostudio) [reply lib/src/store/moor/moor_store.dart]: Todo removed, func renamed

File: lib/src/store/moor/moor_store.dart (hunk 2 above)
Comment (VSemikozInostudio) [reply lib/src/store/moor/moor_store.dart]: FIxed

File: lib/src/store/moor/moor_store.dart [hunk 3]
```diff
@@ -549,7 +547,7 @@ class MoorStore extends Store {
           invitedMembersCount: roomRecord.summaryInvitedMembersCount,
         ),
         directUserId: roomRecord.directUserId != null
-            ? UserId(roomRecord.directUserId ?? '')
+            ? UserId(roomRecord.directUserId!)
```
Comment (MaginMaxim) [inline lib/src/store/moor/moor_store.dart:550]: What if we use just UserId(roomRecord.directUserId ?? '') ?

**Review by MaginMaxim — APPROVED** [review]

File: lib/src/store/moor/moor_store.dart (hunk 3 above)
Comment (VSemikozInostudio) [reply lib/src/store/moor/moor_store.dart:550]: We need to pass null if directUserId is null, not UserId("")

Comment (MaginMaxim) [reply lib/src/store/moor/moor_store.dart:550]: Okey, yes
I agree

**Review by esleptsovaINO — APPROVED** [review]

**Pull request merged**